Add Cloud backup content (#850)
timescale/docs: cloud/backup-restore-cloud.md

any point during that timeframe.  

When you delete an instance, we retain a backup of the instance for seven days.
If you need to restore your database from a backup, [contact support][support].

If you want to verify that your service is being backed up, you can run this
query from the `psql` prompt:
```sql
SELECT pg_is_in_backup()::text
```

## Weekly backups
A full database backup is performed weekly. The two most recent full backups are
automatically saved to a secure Amazon S3 service. For every successful weekly
backup, the logs show `DB BACKED UP`.

If a weekly backup fails to complete, the logs show `DB DIDN'T BACKUP`, with the
time and day that the backup failed to complete. In this case, the backup is
automatically attempted again.

If the backup failed because it took too long to complete, you might need to
increase the CPU size of your system. This can happen if you service has a lot
of data, but not enough CPU processing capability. Increasing the size of the
CPU can help alleviate this bottleneck.

## Daily backups
Incremental database backups are performed daily. This backs up the database

> first against the weekly full backup, and then against the previous days

Reviewer: ```suggestion
first against the weekly full backup, and then against the previous day's
```

incremental backup. This means that each day has a full record that can be used
to restore the service, but does not take as long to run as a full backup.

> Because writes to the database occur in realtime, the service commits

Reviewer: ```suggestion
Because writes to the database occur in real time, the service commits
```

write-ahead log (WAL) segments of all changes by streaming them to an Amazon S3
service. This ensures that any data committed is available for recovery.

> If an incremental backup fails to complete it will try again the next day. In

Reviewer: ```suggestion
If an incremental backup fails to complete, it will try again the next day. In
```

this case, the backup contains the difference between the last complete backup
and the current day.

> ## Restoring from backup

Reviewer 2: ```suggestion
## Restore from backup
```

> If your database fails the restore process begins automatically. This occurs in

Reviewer: ```suggestion
If your database fails, the restore process begins automatically. This occurs in
```

> two distinct stages:

Reviewer: ```suggestion
two stages:
```

1.  Restore: The PostgreSQL service uses `pg_restore` to restore from the most
    recent successful backup. In most cases, the most recent successful backup
    is no more than a day old; either a full weekly backup, or an incremental
    daily backup. How quickly this occurs is limited by the CPU size of the
    service. Large services can take a long time to recover.
1.  Recovery: When the restore is complete, recovery can begin. This is where
    the database replays the WAL segments that have been created since the last
    successful backup. This stage is processed using a single thread, so the

>     speed of this stage is not dependent on the size of the service, but the

Reviewer: ```suggestion
    speed of this stage is dependent not on the size of the service, but on the
```
